.travis.yml

```diff
@@ -5,6 +5,7 @@ sudo: false
 addons:
   apt_packages:
     - graphviz
+    - ruby2.1
```

Comment:
This is needed just for deploying right? Maybe there is a way of installing only when deploying ?